docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,95 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+Signals are similar to ops, but are not stored; they're useful to implement features like user presence. They provide a simple way to communicate transient data to connected clients within a Fluid application.
+
+They are lossy and not guaranteed to be ordered on delivery relative to other signals and ops. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol.
+
+## Why are signals useful?
+Signals provide a transient data channel for sharing short-lived information that does not need to be persisted in the Fluid op stream.
+
+By sending signals, you avoid the storage and sequencing of data that will not be relevant or useful in the long-term.
+
+Signals are the most appropriate data channel in many user presence scenarios, where each user has the responsibility of sharing their current presence state to other connected users. In these scenarios, current presence data is short-lived, past presence state is irrelevant, and the shared data is not persisted on disconnect.
+
+## How can I use signals in Fluid?
+The `Signaler` DataObject can be used to effectively integrate and send communications via signals in a Fluid application. `Signaler` allows clients to efficiently send signals to other connected clients and add/remove listeners for specified signal types.
+
+# Signaler
+
+## Usage
+User presence scenarios are well-suited for `Signaler`, as users are required to tell other users their own information and their past data is mostly irrelavant. Using `Signaler` over other distributed data structures in these scenarios is beneficial, as its usage does not result in the storage of data that is not useful in the long-term.
```

Comment:
Probably keep this section ("Signaler" -> "Usage") scenario-agnostic as possible, and move anything specific to user presence down in the Common Patterns section

Reply:
Ok cool. Looking at it no, I feel like the "Usage" section is sort of redundant within the context of this article, as everything said is already mentioned in the "Why are signals useful?" section. I think jumping straight into the creation of Signaler after its preamble in the "How can I use signals in Fluid?" would be appropriate.